platforms/chibios/drivers/ws2812_pwm.c

```diff
@@ -79,15 +79,15 @@
 #    endif
 #endif
 
-#ifndef WS2812_PWM_TARGET_PERIOD
-//#    define WS2812_PWM_TARGET_PERIOD 800000 // Original code is 800k...?
-#    define WS2812_PWM_TARGET_PERIOD 80000 // TODO: work out why 10x less on f303/f4x1
```

Comment:
What device are you testing on? I wonder if this comment still applies (if it ever did...).

Reply:
I'm testing on an STM32F411.

I can't conceive of a realistic scenario where 800k wouldn't have worked, but where 80k would work. The precision needed to get the "high time" pulse widths correct requires tick frequencies high enough to support 800k. The extra 10us of waiting while low using the 80kHz version is just wasted time.

Reply:
Ah yeah, when I was cleaning up the WS2812 API I had some issues testing the PWM driver on a Proton-C. zvecr suggested I change it to 800k and it started working. I guess something changed elsewhere since then, possibly related to systick stuff, that rendered this unnecessary.